internal/controller/promotions/promotions.go

```diff
@@ -389,7 +389,7 @@ func (r *reconciler) promote(
 		)
 	}
 	if targetFreight == nil {
-		return nil, fmt.Errorf("Freight %q not found in namespace %q", promo.Spec.Freight, promo.Namespace)
+		return nil, fmt.Errorf("freight %q not found in namespace %q", promo.Spec.Freight, promo.Namespace)
```

Comment:
Even though linters complain about it, "Freight" is on purpose here because it's the name of the object kind (same applies to other places where this was changed).

Reply:
agreed.